plugins/single_plugins/scale.cpp

```diff
@@ -612,7 +621,10 @@ class wayfire_scale : public wf::plugin_interface_t
 
             std::vector<size_t> row;
             x = workarea.x + (int) spacing;
-            width = (workarea.width - (n + 1) * (int) spacing) / n;
+            if (i == lines - 1)
+            {
+                x += (grid_cols - grid_last_row_cols) * width / 2.0;
```

Comment:
```suggestion
                width = (workarea.width - (n + 1) * (int) spacing) / n;
```

I'd rather leave this what it was for the last row.

Reply:
This is in response to SnooSnoo-ui's comment about windows in the last row being bigger. But this can be a separate issue for discussion, so I'm reverting to the previous behavior for now.